data/governments.txt

```diff
@@ -403,6 +403,28 @@ government "Heliarch"
 		"Quarg" -.01
 		"Coalition" 1
 
+government "Hicemus"
+	swizzle 0
+	color 0.6 0.0 0.4
+	"crew attack" .2
+	"crew defense" 1
+	language "Incups"
+	"player reputation" 0
+	"attitude toward"
+		"Quarg" 1
+		"Incipias" 1
+
+government "Incipias"
+	swizzle 0
+	color 0.5 0.0 0.9
+	"crew attack" .2
+	"crew defense" 1
+	language "Incups"
+	"player reputation" 1
+	"attitude toward"
+		"Quarg" 1
```

Comment:
Are the Incipias aware of the Quarg initially? If not, this should be added by event once they encounter the Quarg, rather than being present from the start of the game.

Reply:
They are kind of aware

Reply:
Thats weird, I dont think they should care this much
Otherwise you randomly arrive here and get shot at

Reply:
ok